composer.json

```diff
@@ -30,5 +31,11 @@
         "lint-clean": "phpcbf code/ *.php"
     },
     "minimum-stability": "dev",
-    "prefer-stable": true
+    "prefer-stable": true,
+	"config": {
+		"allow-plugins": {
+			"composer/installers": true,
+			"silverstripe/vendor-plugin": true
+		}
+	}
```

Comment:
```suggestion
```
Don't need these here, we have allow-plugins in installer + sink, but not other modules